userguide/kaui/includes/overdue.adoc

```diff
@@ -0,0 +1,188 @@
+
+| Block Subscription Changes
+| If set to `true`, the customer can make plan changes to the subscription in this overdue state. If set to `false`, the customer cannot make changes.
+
+| Subscription Cancellation
+a| Select the option that describes how Kill Bill behaves regarding a cancellation in this overdue state:
+
+* `NONE`: The subscription is not cancelled. _Not sure this is right_
+
+* `POLICY_NONE`: ??? _Maybe some to do with the default cancellation policy in the catalog? Not sure..._
+
+* `POLICY_IMMEDIATE_POLICY`: Cancels the subscription immediately and applies a partial credit to the
+account based on how much of the service has been consumed.
+
+* `END_OF_TERM`: Cancels the subscription at the end of the billing period with no refund to
+the customer (i.e., no proration).
+
+| Days Since Earliest Unpaid Invoice
+| Specifies how many days should pass after the last unpaid invoice before initiating this overdue flow.
+
+| Tag Inclusion
+a| Specifies that the overdue flow will occur if the account is tagged with one of the selected system (control) tags. To disregard tag inclusion for this overdue state, leave as the default `NONE` setting.
+
+* `AUTO_PAY_OFF`
+* `AUTO_INVOICING_OFF`
+* `OVERDUE_ENFORCEMENT_OFF`
+* `MANUAL_PAY`
+* `TEST`
+* `PARTNER`
+
+For a description of system tags, see the https://docs.killbill.io/latest/userguide_subscription.html#components-tag["Tags"] section of the _Subscription Guide_.
+
+| Tag Exclusion
+| Specifies that the overdue flow will _not_ occur if the account is tagged with one of the selected system (control) tags. See tags information above.
+
+| Number of Unpaid Invoices
+| Specifies that the overdue flow will occur if the number of unpaid invoices equals or exceeds the specified value.
+
+| Total Unpaid Invoice Balance
+| Specifies that the overdue action will occur if the total unpaid invoice balance equals or exceeds the specified value.
+
+|===
+
+=== Upload an XML Overdue Configuration File
+
+. <<View Overdue Configuration,Go to the Overdue Show tab>> on the Tenant Configuration page.
++
+image::OverdueTab.png[width=850,align="center"]
++
+. Click the plus sign ( image:i_PlusGreen.png[] ) next to *Existing Overdue Config*.
++
+Kaui opens the Overdue Configuration screen:
++
+image::OverdueConfig-Empty.png[width=850,align="center"]
++
+. Click *Enable Advanced Configuration (Upload XML)*. Kaui displays an upload screen:
++
+image::Overdue-AdvancedConfigUpload.png[width=450,align="center"]
++
+. Click the *Choose File* button, locate the XML file, and select it.
++
+Once you have selected the file, Kaui displays the filename next to the *Choose File* button.
++
+image::Overdue-AdvancedConfigUpload-File.png[width=450,align="center"]
++
+. Click the *Upload* button.
++
+If the upload is successful, Kaui displays a confirmation message along with a list of plans on the Catalog Show tab:
```

Comment:
Adding a screenshot here in case you need one. 
![upload-overdue](https://user-images.githubusercontent.com/85998496/164417885-e8ee91b7-ceeb-4538-a91b-2b38d1237e3b.png)

Reply:
@reshmabidikar - The screenshot above shows the Catalog tab, not the Overdue tab. (I fixed the incorrect reference to the Catalog Show tab.) I'd love a screenshot of the Overdue Show tab with at least two overdue flows displayed.

Reply:
Same comment as above, I took a screenshot after explicitly clicking on the Overdue tab:
![upload-overdue-2](https://user-images.githubusercontent.com/85998496/164649513-764810b4-2f63-4ff4-800f-c1a40200f923.png)

Reply:
I decided to show the Overdue tab and just mention the success message. See page 5 - 6.